jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/ContextFactory.java

```diff
@@ -28,32 +28,37 @@
 import com.ibm.dtfj.java.JavaRuntime;
 
 /**
- * Factory for creating different types of context
- * 
- * @author adam
+ * Factory for creating different types of contexts.
  *
+ * @author adam
  */
 public class ContextFactory {
-	
+
+	private ContextFactory() {
+		/* no instances needed */
+	}
+
 	/**
-	 * Create a DTFJ context
-	 * 
+	 * Create a DTFJ context.
+	 *
 	 * @param major DTFJ API major version to be supported
 	 * @param minor DTFJ minor version to be supported
 	 * @param space address space for this context (cannot be null)
-	 * @param process in this address space
-	 * @param rt Java runtime for this context (may be null)
+	 * @param process process in this address space
+	 * @param runtime Java runtime for this context (may be null)
 	 * @return the context
 	 */
-	public static IDTFJContext getContext(final int major, final int minor,final Image image, final ImageAddressSpace space, final ImageProcess proc, final JavaRuntime rt) {
-		DTFJContext ctx = new DTFJContext(major, minor, image, space, proc, rt);
+	public static IDTFJContext getContext(final int major, final int minor,final Image image, final ImageAddressSpace space, final ImageProcess process, final JavaRuntime runtime) {
```

Comment:
`@param image` is missing.

Reply:
Thanks. I'll fix that.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/17077/commits/05fa5024114cb40275bd54d22224e85bcf21467d.